### DISCUSSION TITLE
Add the possibility to change the value of error messages in the basicAuthenticationView.tsx

### DISCUSSION THREAD

**Discussion by Leander-Wendt** [iavofficial/IAVFrontendFramework#29] [Ideas]
Currently, the error messages in the [basicAuthenticationView](
https://github.com/iavofficial/IAVFrontendFramework/blob/develop/src/components/authentication/default/basicAuthenticationView.tsx) are hard coded.  
It would be great if you'd be able to pass some values in the props to set them to a custom value.

**Comment by mikewegele** [comment]
Thanks for the input leander!
We have taken this on board and will work on it.

**Comment by mikewegele** [comment]
Hi Leander, this feature is now released! https://github.com/iavofficial/IAVFrontendFramework/releases/tag/v1.3.0
